jfrog-cli/artifactory/commands/golang/go.go

```diff
@@ -93,3 +104,24 @@ func validatePrerequisites() error {
 	}
 	return nil
 }
+
+/*
+Resolves the dependencies from the official Go repositories and publish those dependencies to Artifactory
```

Comment:
Let's use // instead of /*...*/ for comments, to be consistent with the existing code.
There are a few other such comments in this PR which I suggest fixing.

The method should named as a private method, because it is not used outside this file. I renaming it to *downloadAndPublish*.

I suggest rephrasing the function comment to:
*Download the dependencies from VCS and publish them to Artifactory.*

Reply:
Done